inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -1472,7 +1475,21 @@ format layout_optimizer::get_preferred_format(program_node& node) {
             layout{ data_types::f32, format::bfyx, tensor{} }).format;
     } else if (node.is_type<quantize>()) {
         auto layout = node.get_output_layout();
-        if (layout.format.spatial_num() == 2 &&
+
+        std::function<bool(const program_node& node)> only_gemm_users = [&](const program_node& node) {
+            std::cout << node.get_users().front()->id() << std::endl;
+            if (node.get_users().size() != 1)
```

Comment:
What if quantize result goes to multiple gemm layers? Maybe we should check that all users are gemm here? Also, what about FC? Don't we need the same for it?

Reply:
yeah I will update to be applied to such cases. I will check FCs too whther they are not supporting blocked format too.

Reply:
Updated.